README.md

```diff
@@ -88,12 +88,23 @@ To use this bot, you will need to create a Discord bot and invite it to your ser
 
 To start the bot, run the following command:
 
-```
+```bash
```

Comment:
_nitpick:_ I'm not certain I care for the "bash" added here, but I'm not going to block the merge due to it.